apps/portals/package.json

```diff
@@ -10,6 +10,7 @@
     "@mui/material": "^5.14.17",
     "@react-hookz/web": "^23.1.0",
     "@sage-bionetworks/react-base-table": "^1.13.4",
+    "@sage-bionetworks/synapse-types": "workspace:*",
```

Comment:
This package is not exclusively types and does have some JS code, such as enums, so `dependencies` is probably more appropriate than `devDependencies`. This should not impact builds.